ide/editor.completion/src/org/netbeans/modules/editor/completion/CompletionLayout.java

```diff
@@ -367,48 +367,82 @@ private void show(List data, String title, int anchorOffset, Point location,
                 completionScrollPane = new CompletionScrollPane(
                     editorComponent, listSelectionListener,
                     new MouseAdapter() {
+                        CompletionItem selectedItemOnPress;
+
+                        @Override
+                        public void mousePressed(MouseEvent evt) {
+                          if (!SwingUtilities.isLeftMouseButton(evt)) {
```

Comment:
wouldn't
if (SwingUtilities.isLeftMouseButton(evt)) {
        selectedItemOnPress = completionScrollPane.getSelectedCompletionItem();
}
be more readable?